styleguide/_themes/derek/scss/components/_footer.scss

```diff
@@ -1,3 +1,77 @@
+a.footer-menu-link {
+  color: darken($light-gray, 15%);
+  font-size: 1.2rem;
+}
+a.footer-tab-link{
+  color:$white;
+  font-size: 1.3rem;
+}
 .footer{
-  
-}
\ No newline at end of file
+  ul{
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+}
+.footer-menu {
+  li {
+    list-style: none;
+    padding: 0;
+    margin: 8px 0;
+  }
+  &.split {
```

Comment:
Maybe create a `.footer-menu-split` class instead and take out of here.  `.split` is pretty generic, so it'd be ideal to have something a bit more specific to the context.